mhkit/loads/extreme.py

```diff
@@ -727,3 +727,31 @@ def mler_export_time_series(rao, mler, sim, k):
     mler_ts = mler_ts.rename(columns={0: 'WaveHeight', 1: 'LinearResponse'})
 
     return mler_ts
+
+
+def return_year_value(ppf, return_year, short_term_period_hr):
+    """
+    Calculate the value from a given distribution corresponding to a particular
+    return year.
+
+    Parameters
+    ----------
+    ppf: callable function of 1 argument
+        Percentage Point Function (inverse CDF) of short term distribution.
+    return_year: int
```

Comment:
Does ``return_year`` have to be an int? I think we could allow floats here as well without issue.

Reply:
Good point, I agree that it could be a float. Updated.